superset/models/sql_lab.py

```diff
@@ -224,10 +225,10 @@ def data(self) -> Dict[str, Any]:
         for col in self.columns:
             column_name = str(col.get("column_name") or "")
             order_by_choices.append(
-                (json.dumps([column_name, True]), column_name + " [asc]")
+                (json.dumps([column_name, True]), column_name + __(" [asc]"))
```

Comment:
Wondering whether the space should be translated or not here, too (and just below)